plugins/net_plugin/net_plugin.cpp

```diff
@@ -3178,18 +3178,6 @@ namespace eosio {
    void net_plugin_impl::plugin_shutdown() {
          in_shutdown = true;
 
-         connections.stop_conn_timers();
-         {
-            fc::lock_guard g( expire_timer_mtx );
-            if( expire_timer )
-               expire_timer->cancel();
-         }
-         {
-            fc::lock_guard g( keepalive_timer_mtx );
-            if( keepalive_timer )
-               keepalive_timer->cancel();
-         }
-
```

Comment:
Didn't see a reason for these cancellations. The timers will effectively be stopped once the named_thread_pool is stopped.